core/modules/filters/debug-log-js

```diff
@@ -0,0 +1,36 @@
+/*\
+title: $:/core/modules/filters/debug-log.js
+type: application/javascript
+module-type: filteroperator
+
+Filter operator for logging the current list and an optional variable value to the console
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+/*
+Export our filter function
+*/
+exports["debug-log"] = function(source,operator,options) {
+	var results = [], data = {};
+	
+	if (operator.operand)
+		data[operator.operand] = options.widget.getVariable(operator.operand,{defaultValue:""});
+
+	source(function(tiddler,title) {
+		results.push(title);
+	});
+	
+	console.group("filter-debug");
```

Comment:
IMO if new logging functionality is needed, $:/core/modules/utils/logger.js needs to be extended. IMO `console` could be "undefined"

Reply:
Yes, seems we can use core logger here, and add new group method to the core logger.

Reply:
Yes, seems we can use core logger here, and add new group method to the core logger.

Reply:
@yaisog will you do this? Do you need help on this?

Reply:
I don't think I can do this. Maybe with some help? I have no idea what this is about.